docs/_automation/powershell/README.md

```diff
@@ -34,10 +34,22 @@ The FinOps toolkit PowerShell module is a collection of commands to automate and
 
 ## 📥 Install the module
 
+The FinOps toolkit module requires PowerShell 7, which is supported on all major operating systems but is not installed by default. For details, see [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell). Once PowerShell has been installed, run the following:
+
 ```powershell
+Install-Module -Name Az.Accounts
+Install-Module -Name Az.Resources
 Import-Module -Name FinOpsToolkit
```

Comment:
````suggestion
```

To install the FinOps toolkit module, run the following in either Azure Cloud Shell or a PowerShell client:

```powershell
Import-Module -Name FinOpsToolkit
````